### ISSUE TITLE
[Query Language] for retrieving blocks

### ISSUE THREAD

**Issue by stdword** [stdword/logseq13-full-house-plugin#54]
Similar to [Page Query Language](https://stdword.github.io/logseq13-full-house-plugin/#/reference__query_language), but for blocks.

Could be used for tasks like: getting random block, getting blocks with specified refs, getting blocks under certain parent, agile filtering of blocks, etc.

### If you are waiting for this feature to be available:
- Please add 👀 reaction so I can prioritise it over other features.
- [Subscribe](https://stackoverflow.com/questions/14140564/how-to-follow-only-specific-issues-on-github-instead-of-watching-an-entire-proj/14140659#14140659) for notifications on this issue to keep track of it's progress.

**Comment by samuelukern** [comment]
Yes. Getting a random block with a specified tag would be amazing.
